src/help/contribution/README.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Contribute to Paperback
+lang: en-US
+---
+
+:::guide Contribute to Paperback
+Welcome to Paperback contributing guide. If Paperback initially is a closed source application, it involves a pretty important open source ecosystem. Everyone is welcome to contribute and help in our various project!
```

Comment:
```suggestion
Welcome to Paperback contribution guide. While the Paperback iOS application is a closed source application, it involves a pretty important open source ecosystem. Everyone is welcome to contribute and help in our various project!
```

Reply:
```suggestion
Welcome to the Paperback contribution guide. While the Paperback app is closed source, the open source ecosystem forms a crucial part of the app. Everyone is welcome to contribute and help on the various projects.
```

I feel this is better.

Reply:
I would personally prefer Vibrant's "it involves a pretty important open source ecosystem." as the ecosystem is not restricted to the application. There is indeed a website, a backup converter a future paperback-web that are open source projects but not part of the app nor needed and used by it.

For "help in our various project!" or "help on the various projects." I honestly don't know. Using "our" is a bit more personal: our teams are invested in the project, while "the" has a nice neutral tone. A bit more reflection is needed. What would you think?